client/gutenberg/extensions/markdown/editor.js

```diff
@@ -13,52 +13,52 @@ import './editor.scss';
 import edit from './edit';
 import save from './save';
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
-import JetpackBlockType from 'gutenberg/extensions/presets/jetpack/utils/jetpack-block-type';
+import registerJetpackBlock from 'gutenberg/extensions/presets/jetpack/utils/register-jetpack-block';
 
-const MarkdownBlock = new JetpackBlockType( {
-	name: 'markdown',
-	requiredBlock: 'markdown',
-	config: {
-		title: __( 'Markdown' ),
+export const name = 'jetpack/markdown';
 
-		description: (
-			<Fragment>
-				<p>{ __( 'Use regular characters and punctuation to style text, links, and lists.' ) }</p>
-				<ExternalLink href="https://en.support.wordpress.com/markdown-quick-reference/">
-					{ __( 'Support reference' ) }
-				</ExternalLink>
-			</Fragment>
-		),
+export const required = 'markdown';
```

Comment:
Is a single string sufficient? Are there blocks that depend on multiple modules, like `json-api` and `related-posts`?